docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc

```diff
@@ -367,6 +369,9 @@ SECOND 0-59 +
 FRACTION(n) 0-999999 +
 in which n is the number of significant digits (default is 6; minimum is 1; maximum is 6); +
 stored in 2, 4, or 8 bytes depending on number of digits^2^
+.2+| LOB Data Types .2+| LOB | BLOB | Binary Large Object .2+| 10G – no limit + 
+Currently it sets to 10G (default) and it’s controlled by the `CQD LOB_MAX_SIZE`, the size can be changed depending on your needs.

```

Comment:
Possible word smith: "Currently the limit defaults to 10Gb. This limit can be configured as needed using 'CQD LOB_MAX_SIZE'.

Reply:
OK, thanks Dave. :)